### PR TITLE
Add HURUmap Zambia

### LINKED ISSUE #18: [ZM] Include HURUmap.Zambia here
We're consolidating these separate repos into one so lets' add the [HURUmap.Zambia](https://github.com/CodeForAfricaLabs/HURUmap.Zambia) files here following how it's been done for ke and tz.

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

- Add HURUmap Zambia to this repo
- Improve HURUmap Zambia to use `geo_versions`

Fixes #18 

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
screencapture-localhost-8000-1510226617144
screencapture-localhost-8000-profiles-province-2-copperbelt-1510226664513
screencapture-localhost-8000-profiles-province-8-north-western-1510226637707

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: hurumap_ke/templates/profile/profile_detail.html [hunk 1]
```diff
@@ -99,8 +99,6 @@ <h2 class="header-for-columns"><a class="permalink" href="#age" id="age">Age <i
             <div class="column-third">
                 {% include 'profile/_blocks/_stat_list.html' with stat=households.piped_water stat_type='percentage' %}
             </div>
-            <div class="column-two-thirds" id="chart-histogram-households-water_source_distribution" data-stat-type="scaled-percentage"
-                data-chart-title="Households by main source of water"></div>
```
Comment (DavidLemayian) [inline hurumap_ke/templates/profile/profile_detail.html]: Why is this removed?

File: hurumap_ug/profiles/census.py [hunk 2]
```diff
@@ -17,9 +17,15 @@
...
-        'sex', geo, session,
-        table_fields=['sex'])
+    try:
+        sex_dist_data, total_pop = get_stat_data(
+            'sex', geo, session,
+            table_fields=['sex'])
+
+    except LocationNotFound:
+        sex_dist_data = LOCATIONNOTFOUND
+        total_pop = 0
```
Comment (DavidLemayian) [inline hurumap_ug/profiles/census.py:27]: Looks like someone slid this into the PR?

File: hurumap_ke/templates/profile/profile_detail.html (hunk 1 above)
Comment (phillipahereza) [reply hurumap_ke/templates/profile/profile_detail.html]: @DavidLemayian this was removed accidentally. 😞
